data/matchingData/flatFare.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
The type here will need to be 'flatFare' rather than 'periodMultipleServices' to mimic the site behaviour

Reply:
Oops - now fixed!